hub/modules/config/configurations.bal

```diff
@@ -47,7 +47,7 @@ public configurable decimal POLLING_INTERVAL = 10;
 public configurable decimal GRACEFUL_CLOSE_PERIOD = 5;
 
 # The port that is used to start the hub
-public configurable int HUB_PORT = 9000;
+public configurable int HUB_PORT = 9191;
```

Comment:
Why are we changing the hub port ?

Reply:
we use this as default there in our all env

Reply:
resolved